ecl/hqlcpp/hqlcset.cpp

```diff
@@ -835,6 +835,13 @@ IHqlExpression * InlineLinkedDictionaryCursor::getFirstSearchValue(IHqlExpressio
     return matched;
 }
 
+BoundRow * InlineLinkedDictionaryCursor::buildIterateLoop(BuildCtx & ctx, bool needToBreak)
```

Comment:
Yes - when generating the compare function for something containing a dictionary it was generating an internal error.